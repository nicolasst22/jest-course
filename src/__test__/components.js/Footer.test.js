import React from 'react';

import { mount } from 'enzyme';
import renderer from 'react-test-renderer';
import Footer from '../../components/Footer';

describe('Footer', () => {

  const footer = mount(<Footer />);
  test('Render footer', () => {
    expect(footer.length).toEqual(1);
  });
  test('Render del titulo', () => {
    expect(footer.find('.Footer-title').text()).toEqual('Platzi Store');
  });
});

describe('Footer snapshot', () => {
  test('Comprobar render de footer', () => {
    const footer = renderer.create(<Footer />);
    expect(footer.toJSON()).toMatchSnapshot();
  });
});

