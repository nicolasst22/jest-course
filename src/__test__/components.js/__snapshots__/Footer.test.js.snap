// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot Comprobar render de footer 1`] = `
<div
  className="Footer"
>
  <p
    className="Footer-title"
  >
    Platzi Store
  </p>
  <p
    className="Footer-copy"
  >
    Todos los Izquierdos Reservados
  </p>
</div>
`;
